frontend/www/js/omegaup/components/arena/ContestListv2.vue

```diff
@@ -64,9 +88,13 @@ export enum ContestTab {
 })
 export default class ArenaContestList extends Vue {
   @Prop() contests!: types.ContestList;
+  @Prop() initialQuery!: string;
```

Comment:
```suggestion
  @Prop() query!: string;
```

uno de estos días por fin vamos a terminar de deprecar todos los `initial...` para que ya no exista este patrón en ningún lugar.